### PR TITLE
[RND201-187] change import paths for better readibility

### PR DESCRIPTION
**Pull request by Anna-Majda-Mobica**
Updating paths + auto formatting all the files

### CODE REVIEW

File: tests/src/page_objects/keywords/home_page.robot [hunk 1]
```diff
@@ -1,83 +1,84 @@
...
+    Click Element    ${StartButton}
+    Log    Click Start Button    console=yes
     Wait Until Element Is Enabled    ${StartButton}
-    
+
 Log Console Is Visible
-    Wait Until Element Is Visible   ${CollapseLogsButton}
-    ${variable}    Run Keyword And Return Status    Element Should Be Visible   ${LogsTitle}
+    Wait Until Element Is Visible    ${CollapseLogsButton}
+    ${variable} =    Run Keyword And Return Status    Element Should Be Visible    ${LogsTitle}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/home_page.robot]: You removed "=" sign in other places but left one here

File: tests/src/page_objects/keywords/home_page.robot [hunk 2]
```diff
@@ -1,83 +1,84 @@
...
     Wait Until Element Contains    ${LogsConsole}    ${message}    ${timeout}
-    
+
 Check If Log Console Does Not Contain Errors
-    Log    Check if there are no errors or warnings in the Log Console   console=yes
-    ${logs}=    Get Text    ${LogsConsole}
-    @{log_list}=    Split To Lines    ${logs}
+    Log    Check if there are no errors or warnings in the Log Console    console=yes
+    ${logs} =    Get Text    ${LogsConsole}
+    @{log_list} =    Split To Lines    ${logs}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/home_page.robot]: As above

File: tests/src/page_objects/keywords/home_page.robot [hunk 3]
```diff
@@ -1,83 +1,84 @@
...
     Log    Validate file details in Log Console    console=yes
     Check If Log Console Contains    Total size: ${file}[archive_size_bytes] bytes    10
     Check If Log Console Contains    Done. Creating ZIP file
-    
+
 Validate Output Description
     [Arguments]    ${expected_output}
     Log    Validate Output Description    console=yes
-    ${output} =    Get Text    ${Description}  
+    ${output} =    Get Text    ${Description}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/home_page.robot]: As above

File: tests/src/page_objects/keywords/home_page.robot [hunk 4]
```diff
@@ -1,83 +1,84 @@
...
     Element Should Contain    ${Description}    ${expected_output}
     ...    error=Description validation failed. Actual: ${output}, expected: ${expected_output}
 
 Validate Output Archive Files Number
     [Arguments]    ${files_num}
     Log    Validate archive files number    console=yes
-    ${output_filenum} =    Get Text    ${FileNum} 
+    ${output_filenum} =    Get Text    ${FileNum}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/home_page.robot]: As above

File: tests/src/page_objects/keywords/home_page.robot [hunk 5]
```diff
@@ -1,83 +1,84 @@
...
     Element Should Contain    ${FileNum}    ${files_num}
     ...    error=Validation files number failed. Actual: ${output_filenum}, expected: ${files_num}
 
 Validate Output Archive File Size
     [Arguments]    ${size}
     Log    Validate archive size    console=yes
     ${output_size} =    Get Text    ${FileSize}
-    ${size} =   Convert To Mega    ${size} 
+    ${size} =    Convert To Mega    ${size}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/home_page.robot]: As above

**Review by Mariusz-Poznanski-Mobica — APPROVED** [review]

**Review by krzysztof-rodzaj-mobica — APPROVED** [review]

**Pull request merged**